suite-native/module-passphrase/src/components/PassphraseForm.tsx

```diff
@@ -77,32 +90,47 @@ export const PassphraseForm = ({ inputLabel, onFocus }: PassphraseFormProps) =>
         dispatch(onPassphraseSubmit({ value: passphrase, passphraseOnDevice: false }));
     });
 
-    const inputHasValue = !!watch('passphrase').length;
+    const handleFocusInput = () => {
+        setIsInputFocused(true);
+        onFocus?.();
+    };
 
     return (
         <Form form={form}>
-            <VStack spacing="medium" padding="medium" style={applyStyle(formStyle)}>
-                <TextInputField
-                    label={inputLabel}
-                    name="passphrase"
-                    maxLength={formInputsMaxLength.passphrase}
-                    accessibilityLabel="passphrase input"
-                    autoCapitalize="none"
-                    onFocus={onFocus}
-                    secureTextEntry
-                />
-                {inputHasValue && (
+            <Card style={applyStyle(cardStyle)}>
+                <VStack spacing="medium" style={applyStyle(formStyle)}>
+                    <TextInputField
+                        label={inputLabel}
+                        name="passphrase"
+                        maxLength={formInputsMaxLength.passphrase}
+                        accessibilityLabel="passphrase input"
+                        autoCapitalize="none"
+                        onFocus={handleFocusInput}
+                        onBlur={() => setIsInputFocused(false)}
+                        secureTextEntry
+                    />
                     <Animated.View entering={FadeIn} exiting={FadeOut}>
-                        <Button
-                            accessibilityRole="button"
-                            accessibilityLabel="confirm passphrase"
-                            onPress={handleCreateHiddenWallet}
-                        >
-                            <Translation id="modulePassphrase.form.enterWallet" />
-                        </Button>
+                        {isDirty && (
+                            <Button
+                                accessibilityRole="button"
+                                accessibilityLabel="confirm passphrase"
```

Comment:
thinking outloud (haha, I know I wrote it): Do you think we should use intl for accessibilityLabels or not?

Reply:
I personally wouldn't as even this in my opinion is redundant. We discussed this a while ago and this makes sense to add to be compliant somewhat with the accessibility principles, but Trezor devices will never be accessible by nature, so it's overkill to translate this IMO. But we can discuss as I'm not sure and never thought about this too much :D

Reply:
i thought a bit about this over the weekend and in the end i also tend not localise, since trezor is ot very accessible device anyway.